EIPS/eip-7775.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 7775
+title: BURN opcode
+description: An opcode to burn native token at the given address
+author: Dev Bear (@itsdevbear)
+discussions-to: https://ethereum-magicians.org/t/eip-7775-burn-opcode/21287
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-09-30
+---
+
+## Abstract
+
+This proposal introduces a new `BURN` opcode, designed to burn native tokens at the given address.
+
+## Motivation
+
+The motivation for this proposal is to provide a standardized and efficient way to burn native tokens directly within the EVM. Historically, contracts such as the BeaconDepositContract have "burned" ether by making it irrecoverable from the given address. This approach can lead to confusion and potential misuse. By introducing a dedicated `BURN` opcode, we can ensure a clear and consistent method for burning native tokens. This could become useful for Ethereum L2s when transferring ether back to the L1, as well as other EVM L1 chains that could leverage this for their cryptoeconomics. Additionally, it opens up an interesting design space for [EIP-7685](./eip-7685.md) use cases, providing a more robust and standardized mechanism for token management.
```

Comment:
I do not understand what EIP 7685 here has to do with it. What request is triggered by the EL here which should be forwarded to the CL?

Reply:
Removed, I was more giving an example of what could be an interesting usecase. The `BURN` opcode being utilized within a SystemCall, which could utilize 7685 to relay info to the CL.